packages/ethereum/contracts/src/static/stake/HoprStakeSeason8.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.0;
+import './HoprStakeBase.sol';
+
+/**
+ * Staking program - Season 8
+ * Inherit a HoprStakeBase.
+ * Include new function for owner to stake for a group of accounts
```

Comment:
This function was already included as part of staking season 7 I believe. Therefore, we can delete this sentence?

Reply:
changed to better description.